### DISCUSSION TITLE
Only ❤️ for this library. TypeScript types for API exports maybe?

### DISCUSSION THREAD

**Discussion by strlns** [vikejs/vike#725] [Discussion]
Just wanted to say, almost every time I had a question with this library, I found the answer in the excellent docs.
Only case when it wasn't, I started a discussion and received immediate reply from maintainer @brillout . Great.

Hope this is not too much reddit-like babble, just very satisfied.

Built my small SSG portfolio site and it was much fun, in the end a 100 Lighthouse score on cheap shared hosting despite using component library and React.
Loved the experience, am excited to try out with SSR next.

Next thing I wanted to ask: 
> is it safe to disable client routing for mobile and how to do it? Or why are clicks on links are slow on a fast smartphone?

After skimming the docs for only a few seconds, the solution was already clear, just change the preloading, or even make it dependent on media query – last problem went away as well 💯

Exported member `clientRouting` needs to be constant I guess (edit: no need to guess, there's a helpful error), but that makes a lot of sense.

Small question:
Some more types would be great, of course it's not possible to hint what exports are "magic" without documentation, but the docs are very good here and the API surface seems small and intuitive.

But for those members, maybe there could be more includedntypes, e.g.

```
type PrefetchStaticAssetsValue = {
  when: "HOVER" | "VIEWPORT" | false;
};
```

Maybe it's just me and I didn't find this or this would be considered bloat, for me it would be a nice touch in addition to the good web docs

**Comment by brillout** [comment]
Agreed! It's actually on the roadmap of the V1 design.

Thanks for the kind words. I'm glad you're enjoying it. Btw. you may want to join the Discord; there are some disucssions going on about rebranding VPS to "Vike".

And contributions welcome! If that's something you can see yourself liking.
